fixtures/attribute-behavior/AttributeTableSnapshot.md

```diff
@@ -526,23 +526,23 @@
 ## `amplitude` (on `<feFuncA>` inside `<svg>`)
 | Test Case | Flags | Result |
 | --- | --- | --- |
-| `amplitude=(string)`| (changed)| `<number: 0>` |
```

Comment:
Looked at this one. Appears to be a browser behavior change. Chrome warns in console about invalid value.